packages/create/src/recipes/base/_eslint.config.js

```diff
@@ -0,0 +1,24 @@
+import { FlatCompat } from "@eslint/eslintrc";
+import js from "@eslint/js";
+
+const compat = new FlatCompat();
+
+export default [
```

Comment:
Spicy: going to use the new "flat config" style of ESLint to avoid having to do a big overhaul in the future once ESLint 9 lands with this style on by default.

Open to anyone saying this is a bad idea 😂